### PR TITLE
1804 views not preserved

### LINKED ISSUE #1804: Views not preserved when navigating to/from a viewGroup
Hello team Luigi!

I have defined the viewGroup field for my nodes but iframes are not being reused nonetheless. Is there anything else that needs to be done for this feature to work?
There is an if-condition on a field named 'viewgroup' at [core/src/App.html#L293](https://github.com/SAP/luigi/blob/ab58877aedb936e75e00e5b51494c2451b0b0bf3/core/src/App.html#L293) but this field doesn't seem to be assigned anything anywhere... Maybe it's a bug?

And a side question; Is it possible to access "preservedView"s using the Luigi.navigation().navigate method and NOT the back button/method?

### PR DESCRIPTION
**Pull request by legteodav**
**Related issue(s)**
`Fixes #1804`

### CODE REVIEW

File: docs/navigation-parameters-reference.md [hunk 1]
```diff
@@ -199,7 +199,40 @@ settings: {
 
 ### viewGroup
 - **type**: string
-- **description**: allows you to associate nodes to be rendered in the same iframe, as long as they belong to the same origin. The value of this parameter is considered as the view group id. For further explanations, see [this section](navigation-configuration.md#view-groups).
+- **description**: allows you to associate nodes to be rendered in the same iframe, as long as they belong to the same origin. 
+The value of this parameter is considered as the view group id. If you want to use this feature, you must configure [preloadViewGroups](navigation-advanced.md#preloadViewGroups) 
+(value true) and provide a preload ulr for you view group id in setting  [viewGroupsSettings](navigation-advanced.md##viewGroupsSettings).
```
Comment (alexandra-simeonova) [inline docs/navigation-parameters-reference.md:204]:
```suggestion
The value of this parameter is considered as the view group ID. If you want to use this feature, you must set [preloadViewGroups](navigation-advanced.md#preloadViewGroups) to `true` and provide a preload URL for your view group ID in [viewGroupsSettings](navigation-advanced.md##viewGroupsSettings).
```

File: docs/navigation-parameters-reference.md [hunk 2]
```diff
@@ -199,7 +199,40 @@ settings: {
...
+For further explanations, see [this section](navigation-configuration.md#view-groups). 
+- **example**
```
Comment (alexandra-simeonova) [inline docs/navigation-parameters-reference.md:206]:
```suggestion
- **example:**
```

File: test/e2e-test-application/src/app/project/view-group/view-group.component.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    this.config = JSON.stringify({
+      viewGroupSettings: this.window.parent['Luigi'].config.navigation.viewGroupSettings,
+      preloadViewGroups: this.window.parent['Luigi'].config.navigation.preloadViewGroups
+    }, null, 3);
+
+    // this.id = iframes[0]['luigi'].id;
+    // let luigiMf = iframes[0]['luigi'];
+    // const clonedLuigiMf = Object.assign({}, luigiMf);
+    // clonedLuigiMf.currentNode =  Object.assign({}, luigiMf.currentNode);
+    // delete clonedLuigiMf.currentNode.parent;
```
Comment (ndricimrr) [inline test/e2e-test-application/src/app/project/view-group/view-group.component.ts:39]: I guess we can delete if not needed 
```suggestion
```

**Review by ndricimrr — COMMENTED** [review]
Aside from the docu and some comments the example LGTM 👍

**Review by ndricimrr — APPROVED** [review]
Looks good 👍

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**